lib/slappy/listeners/concerns/validatable.rb

```diff
@@ -29,6 +29,7 @@ def valid?(event)
       private
 
       def time_valid?(event)
+        return true unless event.has_key?(:ts)
```

Comment:
Style/DeprecatedHashMethods: `Hash#has_key?` is deprecated in favor of `Hash#key?`.